modules/core/23-commitment/types/merkle.go

```diff
@@ -234,16 +226,8 @@ func (proof *MerkleProof) Empty() bool {
 	return proof == nil || proto.Equal(proof, blankMerkleProof) || proto.Equal(proof, blankProofOps)
```

Comment:
`blankProofOps` check? This can be deleted right?? I don't understand the context of when one could provide that type as `MerkleProof`?

Reply:
Agree, doesn't seem to make sense to me